Makefile

```diff
@@ -7,7 +7,7 @@ SPHINXBUILD   = sphinx-build
 SOURCEDIR     = source
 BUILDDIR      = build
 LINTER        = doc8
-LINTEROPTS    = --ignore D001 # D001 is linelength
+LINTEROPTS    = --ignore D001 --ignore D002 --ignore D004 # Ignore line length and line endings.
```

Comment:
We need to check line endings somewhere. Otherwise, there will be PRs that change the line endings.